apps/site/components/Common/AvatarGroup/Avatar/index.tsx

```diff
@@ -11,16 +13,18 @@ export type AvatarProps = {
 
 const Avatar: FC<AvatarProps> = ({ src, alt, fallback }) => (
   <RadixAvatar.Root className={styles.avatarRoot}>
-    <RadixAvatar.Image
-      loading="lazy"
-      src={src}
-      alt={alt}
-      title={alt}
-      className={styles.avatar}
-    />
-    <RadixAvatar.Fallback delayMs={500} className={styles.avatar}>
-      {fallback}
-    </RadixAvatar.Fallback>
+    <Link href={`https://github.com/${alt}`}>
```

Comment:
On some pages, such as `/blog` page, user names are sent as `alt`, this may cause incorrect links to be created 🤔

Reply:
good call. i think we should continue to pursue #7143 instead